project_task_materials_stock/models/project.py

```diff
@@ -0,0 +1,151 @@
+# -*- coding: utf-8 -*-
+# (c) 2015 Antiun Ingeniería S.L. - Sergio Teruel
+# (c) 2015 Antiun Ingeniería S.L. - Carlos Dauden
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+from openerp import models, fields, api
+from openerp.tools.float_utils import float_round
+
+
+class ProjectTaskType(models.Model):
+    _inherit = 'project.task.type'
+
+    consume_material = fields.Boolean(
+        string='Consume Material',
+        help="""If you select this option, when a task is in this state
```

Comment:
Rephrase it to: "If you mark this check, when a task goes to this state, it will consume the associated materials"